agnocomplete/urls.py

```diff
@@ -5,9 +5,6 @@
 from .views import AgnocompleteView, CatalogView
 
 urlpatterns = [
-    path(
-        r'^(?P<klass>[-_\w]+)/$',
-        AgnocompleteView.as_view(),
-        name='agnocomplete'),
-    path(r'^$', CatalogView.as_view(), name='catalog'),
+    path('<klass>/', AgnocompleteView.as_view(), name='agnocomplete'),
```

Comment:
Just a question. Why don't we use `re_path()` instead of `path()`

Reply:
Hm i really don't know, but i saw the comments here https://github.com/peopledoc/django-agnocomplete/commit/bcb18379322dab8949469618738bc20de73d3824

But request management already migrated to the Django 3 and didn't faced any problems with previous code (except the warning)